denizen_scripts/survival/repo-link/cuboid_creator.dsc

```diff
@@ -33,14 +37,14 @@ cuboid_command:
   permission message: <&c>Sorry <player.name>, you can't perform that command.
   script:
     - if <context.args.first||null> == null:
```

Comment:
would recommend `<context.args.is_empty>` as opposed to verifying if a tag grabbing the first index of an empty list fails